src/main/resources/react4xp/_entries/ExternalCards.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Link } from '@statisticsnorway/ssb-component-library'
+import PropTypes from 'prop-types'
+import { ExternalLink } from 'react-feather'
+
+const ExternalCards = (props) => {
+  return (
+    <React.Fragment>
+      <div className="container">
+        <div className="row justify-content-start">
+          {props.links.map((link, index) => {
+            return (
+              <div className="external-card mb-4 mr-3" key={index}>
```

Comment:
Columns must be the immediate child of a Row according to the Bootstrap 4.0 grid rules.